ansible/roles/zuul/files/layout.yaml

```diff
@@ -58,7 +60,8 @@ pipelines:
             - ^stable/queens$
             - ^stable/newton$
             - ^stable/ocata$
-            - ^production 
+            - ^production
```

Comment:
I think we should add `$` at the end, so that branch like `production-not-production` are not matched.